pyphare/pyphare/pharein/simulation.py

```diff
@@ -504,6 +504,8 @@ def wrapper(simulation_object, **kwargs):
 
         kwargs["hyper_resistivity"] = check_hyper_resistivity(**kwargs)
 
+        kwargs["strict"] = kwargs.get('strict', True)
```

Comment:
TODO, change default to false, set as "True" in tests

Reply:
TBD?

Reply:
done